tests/model_meta/tests.py

```diff
@@ -251,3 +254,19 @@ def test_relations_related_objects(self):
             ])
         )
         self.assertEqual([field.related_query_name() for field in AbstractPerson._meta._relation_tree], [])
+
+
+class ParentListTests(TestCase):
```

Comment:
> TestCase could be SimpleTestCase or unittest.TestCase (but not sure it's worth the possibility that DB will be used later and not caught)

I thought of it but I kept it this way for the same reason you just mentioned. I think it would be safe to keep it this way for now.

> assets could probably be single lines

Will do.